src/encoded/renderers.py

```diff
@@ -107,19 +108,18 @@ def validate_request_tween(request):
             # Includes page text/html requests.
             return handler(request)
 
-        elif request.content_type != 'application/json':
-            if request.content_type == 'application/x-www-form-urlencoded' and request.path[0:10] == '/metadata/':
-                # Special case to allow us to POST to metadata TSV requests via form submission
-                return handler(request)
+        elif content_type_allowed(request):
```

Comment:
This mechanism was added for ingestion processing in CGAP, but it's harmless and could be useful in Fourfront. It's easier to extend than what's there now.